src/api/app/models/project.rb

```diff
@@ -745,10 +736,16 @@ def flag_status(default, repo, arch, prj_flags, pkg_flags)
     opts[:repository] = repo if repo
     opts[:arch] = arch if arch
     opts[:explicit] = '1' if expl
-    ret = 'enable' if ret == :enabled
-    ret = 'disable' if ret == :disabled
+    ret_str = case ret
```

Comment:
The case could be removed since `flags.status` is [an enum](https://github.com/openSUSE/open-build-service/blob/master/src/api/db/structure.sql#L503). The array below could then be `[ret.to_s, opts]`

Reply:
well, minus the d at the end.

Reply:
Oops. Still the `else` in the case will never be reached.

Reply:
sounds like an elvis is overdue :)

Reply:
the end is necessary. I tried to refactor it to if else condition and it broke some test cases

Reply:
that possibly means ret is already a string - passed as default